cmd/eksctl-anywhere/cmd/deletecluster.go

```diff
@@ -45,23 +42,12 @@ var deleteClusterCmd = &cobra.Command{
 	},
 }
 
-func preRunDeleteCluster(cmd *cobra.Command, args []string) error {
-	cmd.Flags().VisitAll(func(flag *pflag.Flag) {
-		err := viper.BindPFlag(flag.Name, flag)
-		if err != nil {
-			log.Fatalf("Error initializing flags: %v", err)
-		}
-	})
-	return nil
-}
-
 func init() {
 	deleteCmd.AddCommand(deleteClusterCmd)
+	addClusterFlags(deleteClusterCmd, dc.clusterOptions, "filename", "install-packages")
```

Comment:
why do we override filename here? the next line sets that flag

Reply:
I overrode it because the deleteCmd usage differs from create and upgrade cluster. StringVarP panics when I try and just overwrite an existing flag so I wasn't able to just leave it in